mitodl/ocw-hugo-themes
# move filters to course search util

### What are the relevant tickets?
closes https://github.com/mitodl/course-search-utils/issues/67

### Description (What does it do?)
This pr moves facet display functionality from ocw-hugo-themes to course-search util so it can be reused in the open search

The course-search-utils pr is https://github.com/mitodl/course-search-utils/pull/69

### How can this be tested?
Run mit-open

Set
COURSE_SEARCH_API_URL=http://localhost:8063/api/v1/learning_resources_search/
CONTENT_FILE_SEARCH_API_URL=http://localhost:8063/api/v1/content_file_search/

Run `yarn start www`

Go to http://localhost:3000/search/ and click around. Everything should work as expected. The urls will be a little different than the existing ui

Level and department filters should be backward compatible. For example http://localhost:3000/search/?d=Literature should work.

The department facet search filter should work correctly (broken in the parent branch)
